engines/router/missionctl/log/resultlog/proto/turing/TuringResultLog.proto

```diff
@@ -16,11 +16,15 @@ message Request {
 }
 
 message Response {
-    // The JSON response from a Turing component (Enricher / Experiment Engine / Router / Ensembler), UTF-8-encoded.
-    string response = 1;
+    // The JSON response body from a Turing component (Enricher / Experiment Engine / Router / Ensembler),
+    // UTF-8-encoded.
+    string response_body = 1;
```

Comment:
Renamed `response` to `response_body` because the addition of the new field `response_header` would create some ambiguity as to what `response` would really mean.